spec/support/generators.rb

```diff
@@ -8,7 +10,8 @@ def self.valid_edition(attributes = {})
       update_type:     "major",
       body:            "# Heading",
       publisher_title: Edition::PUBLISHERS.keys.first,
-      user:            User.new(name: "Generated User")
+      user:            user,
+      approvals:       [Approval.new(user: user)],
```

Comment:
I think we should not add any values to this generator unless they are absolutely required to pass validations. This will make it very confusing long term i.e. as a new developer I wouldn't expect a `valid_edition` to have any associated approvals. It also makes test setup slower globally